libs/blocks/merch/merch.js

```diff
@@ -4,6 +4,7 @@ import {
 import { replaceKey } from '../../features/placeholders.js';
 
 export const CHECKOUT_LINK_CONFIG_PATH = '/commerce/checkout-link.json'; // relative to libs.
+export const CHECKOUT_LINK_SANDBOX_CONFIG_PATH = '/commerce/checkout-link-sandbox.json'; // relative to libs.
```

Comment:
is /commerce/checkout-link-sandbox.json really needed? Sounds like duplicating authoring and training effort.